medicines/doc-index-updater/src/models.rs

```diff
@@ -61,6 +61,7 @@ pub struct Document {
 
 #[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
 pub enum FileSource {
+    #[serde(alias = "sentinel")]
```

Comment:
https://docs.rs/serde-aux/0.6.1/serde_aux/container_attributes/fn.deserialize_struct_case_insensitive.html
might be able to use this for full case-insensitivity but for now title-case and lowercase covers what we've been doing 👍